clustering/web/hotrod/src/main/java/org/wildfly/clustering/web/hotrod/sso/coarse/CoarseSessionsFactory.java

```diff
@@ -65,6 +68,7 @@ public Map.Entry<String, Map<D, S>> findEntryContaining(S session) {
 
     @Override
     public boolean remove(String id) {
-        return this.cache.withFlags(Flag.FORCE_RETURN_VALUE).remove(new CoarseSessionsKey(id)) != null;
+        this.cache.withFlags(this.ignoreReturnFlags).remove(new CoarseSessionsKey(id));
```

Comment:
@pferraro So using Flag.FORCE_RETURN_VALUE was wrong; i.e. this bit isn't just about including Flag.SKIP_LISTENER_NOTIFICATION?

Reply:
We don't actually need the return value here, so, yes, this is an additional optimization.